### DISCUSSION TITLE
Birthday cmd

### DISCUSSION THREAD

**Discussion by Jeanyvon22** [eritislami/evobot#500] [Ideas]
hello, I was trying to add a birthday function
To automatically wish a member's birthday on their birthday.
I tried on my side, and I come up with something almost functional.
I still have some trouble identifying the next birthday with a "nextbirthday" cmd for expl.
All data is stored on MongoDB.

I think it might be a good idea to add this option back to your bot;)

**Comment by wolf-yuan-6115** [comment]
This is not important for the music bot
